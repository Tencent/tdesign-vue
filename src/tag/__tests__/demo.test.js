/**
 * 该文件为由脚本 `npm run test:demo` 自动生成，如需修改，执行脚本命令即可。请勿手写直接修改，否则会被覆盖
 */

import { mount } from '@vue/test-utils';
import baseVue from '@/src/tag/_example/base.vue';
import deleteVue from '@/src/tag/_example/delete.vue';
import disabledVue from '@/src/tag/_example/disabled.vue';
import iconVue from '@/src/tag/_example/icon.vue';
import longTextVue from '@/src/tag/_example/long-text.vue';
import plainVue from '@/src/tag/_example/plain.vue';
import selectableVue from '@/src/tag/_example/selectable.vue';
import shapeVue from '@/src/tag/_example/shape.vue';
import sizeVue from '@/src/tag/_example/size.vue';
import themeVue from '@/src/tag/_example/theme.vue';

const mapper = {
  baseVue,
  deleteVue,
  disabledVue,
  iconVue,
  longTextVue,
  plainVue,
  selectableVue,
  shapeVue,
  sizeVue,
  themeVue,
};

describe('Tag', () => {
  Object.keys(mapper).forEach((demoName) => {
    it(`Tag ${demoName} demo works fine`, () => {
      const wrapper = mount(mapper[demoName]);
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
