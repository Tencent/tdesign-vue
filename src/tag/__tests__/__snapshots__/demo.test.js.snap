// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag Tag baseVue demo works fine 1`] = `
<div
  class="t-space t-space-vertical"
  style="gap: 16px;"
>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--default t-size-m t-tag--dark"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <a
          href="https://www.tencent.com/zh-cn"
          target="_blank"
        >
          <span
            class="t-tag t-tag--default t-size-m t-tag--dark"
          >
            <span>
              超链接
            </span>
          </span>
        </a>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--dark"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--dark"
        >
          <span>
            标签二
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--dark"
        >
          <span>
            标签三
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--dark"
        >
          <span>
            标签四
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--light"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--light"
        >
          <span>
            标签二
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--light"
        >
          <span>
            标签三
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--light"
        >
          <span>
            标签四
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--outline"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--outline"
        >
          <span>
            标签二
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--outline"
        >
          <span>
            标签三
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--outline"
        >
          <span>
            标签四
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--light-outline"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--light-outline"
        >
          <span>
            标签二
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--light-outline"
        >
          <span>
            标签三
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--light-outline"
        >
          <span>
            标签四
          </span>
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`Tag Tag deleteVue demo works fine 1`] = `
<div
  class="t-space t-space-vertical"
  style="gap: 30px;"
>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--dark t-tag--close"
        >
          <span>
            标签一
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--dark t-tag--close"
        >
          <span>
            标签二
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--dark t-tag--close"
        >
          <span>
            标签三
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--dark t-tag--close"
        >
          <span>
            标签四
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--light t-tag--close"
        >
          <span>
            标签一
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--light t-tag--close"
        >
          <span>
            标签二
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--light t-tag--close"
        >
          <span>
            标签三
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--light t-tag--close"
        >
          <span>
            标签四
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--outline t-tag--close"
        >
          <span>
            标签一
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--outline t-tag--close"
        >
          <span>
            标签二
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--outline t-tag--close"
        >
          <span>
            标签三
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--outline t-tag--close"
        >
          <span>
            标签四
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--default t-size-m t-tag--dark t-tag--ellipsis t-tag--close"
        >
          <span
            class="t-tag--text"
            style="max-width: 100px;"
          >
            
      可删除标签可删除标签
    
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--default t-size-m t-tag--dark t-tag--ellipsis t-tag--close"
        >
          <svg
            class="t-icon t-icon-discount"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M7.84 5.01A2 2 0 115 7.84 2 2 0 017.84 5zm-.7.7a1 1 0 10-1.42 1.42 1 1 0 001.41-1.41z"
              fill="currentColor"
              fill-opacity="0.9"
            />
            <path
              clip-rule="evenodd"
              d="M2 7.95V2h5.95a1 1 0 01.7.3l6.14 6.13a1 1 0 010 1.41L9.84 14.8a1 1 0 01-1.41 0L2.29 8.66A1 1 0 012 7.95zM7.95 3l6.13 6.13-4.95 4.95L3 7.95V3h4.95z"
              fill="currentColor"
              fill-opacity="0.9"
              fill-rule="evenodd"
            />
          </svg>
          <span
            class="t-tag--text"
            style="max-width: 100px;"
          >
            
      可删除标签可删除标签
    
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--default t-size-m t-tag--dark t-tag--close t-is-disabled t-tag--disabled"
        >
          <span>
            
      可删除标签
    
          </span>
          <svg
            class="t-icon t-icon-close t-tag__icon-close"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M8 8.92L11.08 12l.92-.92L8.92 8 12 4.92 11.08 4 8 7.08 4.92 4 4 4.92 7.08 8 4 11.08l.92.92L8 8.92z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="tag-block editable"
    >
      <span
        class="t-tag t-tag--default t-size-m t-tag--dark"
      >
        <span>
          <svg
            class="t-icon t-icon-add"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M7.35 8.65v3.85h1.3V8.65h3.85v-1.3H8.65V3.5h-1.3v3.85H3.5v1.3h3.85z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
          
      添加标签
    
        </span>
      </span>
    </div>
  </div>
</div>
`;

exports[`Tag Tag disabledVue demo works fine 1`] = `
<div
  class="t-space t-space-horizontal"
  style="gap: 30px;"
>
  <div
    class="t-space-item"
  >
    <span
      class="check-tag t-tag t-tag--check t-tag--default t-size-m t-is-disabled t-tag--disabled"
    >
      失效标签
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--default t-size-m t-tag--dark t-is-disabled t-tag--disabled"
    >
      <span>
        失效标签
      </span>
    </span>
  </div>
</div>
`;

exports[`Tag Tag iconVue demo works fine 1`] = `
<div
  class="t-space t-space-horizontal"
  style="gap: 16px;"
>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--default t-size-m t-tag--dark"
    >
      <svg
        class="t-icon t-icon-discount"
        fill="none"
        height="1em"
        viewBox="0 0 16 16"
        width="1em"
      >
        <path
          d="M7.84 5.01A2 2 0 115 7.84 2 2 0 017.84 5zm-.7.7a1 1 0 10-1.42 1.42 1 1 0 001.41-1.41z"
          fill="currentColor"
          fill-opacity="0.9"
        />
        <path
          clip-rule="evenodd"
          d="M2 7.95V2h5.95a1 1 0 01.7.3l6.14 6.13a1 1 0 010 1.41L9.84 14.8a1 1 0 01-1.41 0L2.29 8.66A1 1 0 012 7.95zM7.95 3l6.13 6.13-4.95 4.95L3 7.95V3h4.95z"
          fill="currentColor"
          fill-opacity="0.9"
          fill-rule="evenodd"
        />
      </svg>
      <span>
        默认标签
      </span>
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--default t-size-m t-tag--dark"
    >
      <svg
        class="t-icon t-icon-discount"
        fill="none"
        height="1em"
        viewBox="0 0 16 16"
        width="1em"
      >
        <path
          d="M7.84 5.01A2 2 0 115 7.84 2 2 0 017.84 5zm-.7.7a1 1 0 10-1.42 1.42 1 1 0 001.41-1.41z"
          fill="currentColor"
          fill-opacity="0.9"
        />
        <path
          clip-rule="evenodd"
          d="M2 7.95V2h5.95a1 1 0 01.7.3l6.14 6.13a1 1 0 010 1.41L9.84 14.8a1 1 0 01-1.41 0L2.29 8.66A1 1 0 012 7.95zM7.95 3l6.13 6.13-4.95 4.95L3 7.95V3h4.95z"
          fill="currentColor"
          fill-opacity="0.9"
          fill-rule="evenodd"
        />
      </svg>
      <span>
        
    Icon插槽
  
      </span>
    </span>
  </div>
</div>
`;

exports[`Tag Tag longTextVue demo works fine 1`] = `
<div>
  <span
    class="t-tag t-tag--default t-size-m t-tag--dark t-tag--ellipsis"
    title="默认超八个字超长文本标签超长省略文本标签"
  >
    <span
      class="t-tag--text"
      style="max-width: 150px;"
    >
      默认超八个字超长文本标签超长省略文本标签
    </span>
  </span>
</div>
`;

exports[`Tag Tag plainVue demo works fine 1`] = `
<div
  class="t-space t-space-horizontal"
  style="gap: 16px;"
>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--primary t-size-m t-tag--outline"
    >
      <span>
        标签一
      </span>
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--success t-size-m t-tag--outline"
    >
      <span>
        标签二
      </span>
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--default t-size-m t-tag--outline"
    >
      <span>
        标签三
      </span>
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--warning t-size-m t-tag--outline"
    >
      <span>
        标签四
      </span>
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--danger t-size-m t-tag--outline"
    >
      <span>
        标签五
      </span>
    </span>
  </div>
</div>
`;

exports[`Tag Tag selectableVue demo works fine 1`] = `
<div
  class="t-space t-space-horizontal"
  style="gap: 30px;"
>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--check t-tag--default t-size-m t-tag--checked"
    >
      
    选中
  
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--check t-tag--default t-size-m"
    >
      
    未选
  
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--check t-tag--default t-size-m t-is-disabled t-tag--disabled"
    >
      
    禁用
  
    </span>
  </div>
</div>
`;

exports[`Tag Tag shapeVue demo works fine 1`] = `
<div
  class="t-space t-space-vertical"
  style="gap: 16px;"
>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--default t-size-m t-tag--dark"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--dark"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--default t-size-m t-tag--dark t-tag--round"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--dark t-tag--round"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--default t-size-m t-tag--dark t-tag--mark"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--dark t-tag--mark"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`Tag Tag sizeVue demo works fine 1`] = `
<div
  class="t-space t-space-horizontal"
  style="gap: 30px;"
>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--default t-size-s t-tag--dark"
    >
      <span>
        小型标签
      </span>
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--default t-size-m t-tag--dark"
    >
      <span>
        默认标签
      </span>
    </span>
  </div>
  <div
    class="t-space-item"
  >
    <span
      class="t-tag t-tag--default t-size-l t-tag--dark"
    >
      <span>
        大型标签
      </span>
    </span>
  </div>
</div>
`;

exports[`Tag Tag themeVue demo works fine 1`] = `
<div
  class="t-space t-space-vertical"
  style="gap: 16px;"
>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--dark"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--dark"
        >
          <span>
            标签二
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--dark"
        >
          <span>
            标签三
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--dark"
        >
          <span>
            标签四
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--light"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--light"
        >
          <span>
            标签二
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--light"
        >
          <span>
            标签三
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--light"
        >
          <span>
            标签四
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="t-space-item"
  >
    <div
      class="t-space t-space-horizontal"
      style="gap: 16px;"
    >
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--primary t-size-m t-tag--outline"
        >
          <span>
            标签一
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--success t-size-m t-tag--outline"
        >
          <span>
            标签二
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--warning t-size-m t-tag--outline"
        >
          <span>
            标签三
          </span>
        </span>
      </div>
      <div
        class="t-space-item"
      >
        <span
          class="t-tag t-tag--danger t-size-m t-tag--outline"
        >
          <span>
            标签四
          </span>
        </span>
      </div>
    </div>
  </div>
</div>
`;
